google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/iam/v1/policy.rb

```diff
@@ -121,19 +121,7 @@ class PolicyDelta; end
       #     A single identity requesting access for a Cloud Platform resource.
       #     Follows the same format of Binding.members.
       #     Required
-      class BindingDelta
-        # The type of action performed on a Binding in a policy.
-        module Action
```

Comment:
@eoogbe Could you check why we're no longer generating this enum type and file an issue in `toolkit` if this is unexpected?